migrate/migrations/20240502114803-universal-integration-key.sql

```diff
@@ -0,0 +1,8 @@
+-- +migrate Up notransaction
+ALTER TYPE enum_integration_keys_type
+  ADD VALUE IF NOT EXISTS 'universal';
+
+ALTER TYPE enum_alert_source
+  ADD VALUE IF NOT EXISTS 'universal';
```

Comment:
duplicate
```suggestion
```

Reply:
I was following the pattern the prometheus migration uses where it sets `enum_integration_keys_type` as well as `enum_alert_source`. Is only `enum_integration_keys_type` needed?

Reply:
ohhhhhh, totally missed that they were different. That's embarrassing. don't mind me